drivers/scheduler/sidecars/scripts/wp-install.sh

```diff
@@ -19,6 +19,7 @@ function am_i_first() {
 
 function sleep_forever() {
     log ":: Sleeping forever. Exec into me for debugging"
+    touch /wordpress/alive
```

Comment:
Let's not put the `alive` file on /wordpress since that is a persisted path. Put it on /tmp so on each pod restart, the alive file isn't present.